README.md

```diff
@@ -20,6 +20,15 @@ Then run the flyway tasks, for example:
 ./gradlew flywayMigrate
 </pre>
 
+## Executable Jar
+In addition to using the Gradle plugin the project creates an executable jar that can be used to run migrations.  You will need to pass in the necessary JDBC url and user credental information.
```

Comment:
I assume you create the jar by running ./gradlew build?

Reply:
Yes.  I'll update the documentation.